pytest_fixtures/api_fixtures.py

```diff
@@ -1,7 +1,11 @@
 # Module-wide Nailgun Entity Fixtures to be used by API, CLI and UI Tests
+import logging
+
 import pytest
+from fauxfactory import gen_alphanumeric
```

Comment:
Not a blocker, but you should be able to pass an alphanumeric type to gen_string